Gems/AtomTressFX/Assets/Passes/PassTemplates.azasset

```diff
@@ -0,0 +1,45 @@
+{
+    "Type": "JsonSerialization",
```

Comment:
This file will change name and be included via the Gem code so that no change will be required to the active test PassTemplates.azassets